doc/nrf/releases_and_maturity/migration/2.4.99-cs3_to_2.6.99-cs2/migration_guide_2.4.99-cs3_to_2.6.99-cs2_environment.rst

```diff
@@ -0,0 +1,364 @@
+.. _migration_cs3_to_2_6_99_cs2_env:
+
+Update your development environment for |NCS| v2.6.99_cs2 (for v2.4.99-cs3 users)
+#################################################################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This document describes how to update your development environment from |NCS| v2.4.99-cs3 to |NCS| v2.6.99-cs2.
+
+The main development environment changes introduced by 2.6.99-cs2 for the nRF54H20 DK are the following:
+
+* The |NCS| toolchain has been updated.
+  A bootstrap script specific to the nRF54H20 DK is now available to install and update the toolchain.
+* nRF Util has now replaced nRF Command Line Tools.
+  The bootstrap script will now install and update nRF Util.
+* SDFW and SCFW are now provided as precompiled binaries.
+  The Secure Domain Firmware (SDFW) and System Controller Firmware (SCFW) are no longer built from the source during the application build process, but they must be provisioned as binaries from the provided firmware bundle before the DK can be used.
+  See the details in the `nRF54H20 bring-up`_ section below.
+* The nRF54H20 SoC lifecycle state must now be set to Root of Trust (RoT).
+  See the details in the `Transitioning the nRF54H20 to RoT`_ section.
+
+Minimum requirements
+********************
+
+Make sure you have all the required hardware and that your computer has one of the supported operating systems.
+
+Hardware
+========
+
+* nRF54H20 DK version PCA10175 v0.7.0 or v0.8.0 (ES3).
```

Comment:
0.7.0 or 0.7.2?

Reply:
@miha-nordic is there a 0.7.2? can you confirm?

Reply:
0.7.0 is only ES1 so it will not be distributed. As @hakonfam said 0.7.2 has ES3 on it.